plugin/src/main/resources/org/jenkinsci/plugins/workflow/cps/CpsFlowDefinition/help-script.html

```diff
@@ -1,4 +0,0 @@
-<p>
```

Comment:
whilst this was not the best help - it did at least prompt users where to look?

Reply:
Restored it.